docs/guides/03-custom-deployments.mdx

```diff
@@ -181,7 +181,7 @@ If you are using a mnemonic or a hardware device for your deployer address, refe
 Once the contracts are deployed, the final step is to list the deployment addresses and artifacts in the Lockup docs:
 
 1. List the deployment addresses by opening a PR in the
-   [docs repo](https://github.com/sablier-labs/docs/blob/main/docs/contracts/v2/02-deployments.md)
+   [docs repo](https://github.com/sablier-labs/docs/blob/main/docs/guides/lockup/02-deployments.md)
```

Comment:
Shouldn't this be a relative link?

Reply:
The goal is to point them to the specific file in the docs repo so that they can add the new section and create a PR.